validators/cmd/sigs/sigs-validator_test.go

```diff
@@ -0,0 +1,462 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright the KubeVirt Authors.
+ *
+ */
+
+package main
+
+import (
+	log "github.com/sirupsen/logrus"
+	"gopkg.in/yaml.v3"
+	"kubevirt.io/community/pkg/labels"
+	"kubevirt.io/community/pkg/orgs"
+	"kubevirt.io/community/pkg/sigs"
+	"reflect"
+	"testing"
+)
+
+func Test_validateGroups(t *testing.T) {
```

Comment:
Is the `_` convention? I would expect to see `TestValidateGroups`

Reply:
Hm, good q - I believe that was an autogenerated test func name that I didn't really look at.

Reply:
@xpivarc I investigated a bit - reason for this weird name is that Goland has auto-generated the func name for the non-exported func that is tested.

Now if I rename the func as you requested the IDE will not find the tested func anymore.

Now my q is: does it matter that much? I'd rather expect the test func name to be as close to the name of the func it tests. WDYT?

Reply:
I agree that for _exported_ functions it should be as you said, however for non-exported functions whose name is starting with a lowercase letter the usage of an underscore might be more obvious.

Compare

```go
func DoFuncExported() {}

func TestDoFuncExported() {}
```

to

```go
func doFuncNonExported() {}

func Test_doFuncNonExported() {}
```

where the latter is the matching case here.

Reply:
And again - this is just to help the IDE to find the test for the func and the func for the test func.